README.md

```diff
@@ -1 +1,128 @@
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖  <a name="about-project">Blog App</a>
+   This project is all about a simple blog application where a user can post anything and other users can comment and like the post
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+    <li><a href="https://www.postgresql.org/">PostgreSql</a></li>
+  </ul>
+
+### Key Features <a name="key-features"></a>
+
+- Data Model
+- Medium fidelity UI
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+- Not available -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<ul>
+    <li><a href="https://code.visualstudio.com/">Code editor e.g VS code</a></li>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+      <li><a href="https://rubyonrails.org/">Rails</a></li>
+    <li><a href="https://www.postgresql.org/">PostgreSql</a></li>
+  </ul>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<ul>
+<li>cd my-folder</li>
+<li>git clone https://github.com/Odongo006/My-Blog-App.git</li>
+</ul>
+
+### Install
+
+Install this project with:
+
+<ul>
+<li>bundle install</li>
+</ul>
+
+### Usage
+
+To run the project, execute the following command:
+
+<ul>
+<li>rails server</li>
+</ul>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Kevin Toro**
+- GitHub: [@Odongo006](https://github.com/Odongo006)
+- Twitter: [@HenryOdongo](https://twitter.com/@HenryOdongo007)
+- LinkedIn: [@HenryOdongo](https://www.linkedin.com/in/henry-odongo/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Authentication
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Odongo006/My-Blog-App/issues).
+
+<p align="right">(<a href="#readme-top">back to tops</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give this repo a star and reach out on our social media, and we will be happy to know about it! 😃
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank the Microverse team for their invaluable resources
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/Odongo006/My-Portfolio/blob/main/MIT-LICENSE.txt) licensed.
```

Comment:
- [ Optional ] Please consider fixing this URL to the License file, currently, the link leads to a `404` page not found error. I think it would be better if you create a License file in your project's root directory and then update this link to take a user to that file.